src/backend/src/routes/onboarding.routes.ts

```diff
@@ -10,6 +10,8 @@ onboardingRouter.get('/checklists', OnboardingController.getAllChecklists);
 
 onboardingRouter.get('/checklists/:userId/checked', OnboardingController.getCheckedChecklists);
 
+onboardingRouter.get('/checklists/:userId', OnboardingController.getUsersChecklists);
```

Comment:
this is my fault but instead of passing in userId through the parameters, in the controller you can call req.currentUser and pass that in. You can then change this route to be /checklists/usersChecklists